Libraries/dotNetRDF/Parsing/RdfAParser.NetCore.cs

```diff
@@ -68,7 +68,7 @@ protected override IHtmlDocument LoadAndParse(TextReader input)
 
         protected override bool HasAttribute(IElement element, string attributeName)
         {
-            return !element.HasAttribute(attributeName);
+            return element.HasAttribute(attributeName);
```

Comment:
Good catch. I was wondering why the GoodRelations test started failing all of the sudden